### DISCUSSION TITLE
Frigate Clips

### DISCUSSION THREAD

**Discussion by bm55** [blakeblackshear/frigate#467] [Ask A Question] [answered]
@blakeblackshear Sometimes frigate creates a single long clip, and sometimes it creates several. Occasionally it creates what I think is way too many clips even though it's tracking the objects continuously. I am just curious to know more about when/what causes these multiple clips. 
![image](https://user-images.githubusercontent.com/7528001/103487031-1a44c300-4db7-11eb-85ae-87730f0ab3ae.png)

**Comment by blakeblackshear** [comment]
It creates a clip for each tracked object. Sometimes an object will be lost and detected again with a different ID. That will cause a new clip. There is also a clip per person, so if 2 people walk together, it will create 2 clips.

**Reply by bm55** [reply]
I will experiment with setting longer  max_disappeared. Thanks for the info.

**Comment by bm55** [comment, marked as answer]
@blakeblackshear

**Comment by bmutari** [comment]
I was curious about the same thing, has adjusting the max_disappeared been a fix or workaround for you?
